File: 351001/Orlovich/Dc/Domain/Models/Story.cs
namespace Domain.Models;

public class Story
{
    public long id { get; set; }
    public Editor EditorId {get; set; }
    public string title { get; set; }
    public string content { get; set; }
    public DateTime created { get; set; }
    public DateTime modified { get; set; }
}

File: 351004/Purenok/Task310/src/main/java/com/example/Task310/bean/Story.java
package com.example.Task310.bean;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Story {
    private Long id;
    private Long editorId;
    private String title;
    private String content;
    private LocalDateTime created;
    private LocalDateTime modified;
}